Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.19-alpine3.16@sha256:4e2a54594cfe7002a98c483c28f6f3a78e5c7f4010c355a8cf960292a3fdecfe AS builder
+ARG VERSION=dev-snapshot
+RUN mkdir /build
+RUN apk add --update make
+WORKDIR /build
+ADD . /build
```

Comment:
[nit] I read in Docker's doc that using `COPY` is preferred (see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)

Reply:
done in 9951f226932811c941b7d22593d1e9fd1d849b97